### PR TITLE
Localization stable-4.5 string update

### PR DESCRIPTION
**Pull request by rooftopcellist**
# Description 🛠

Localization Update from 3-28 push.  Soon to follow, the translated strings from the 4-19 push.  

# Reviewer Checklists  👀
Developer reviewer:
- [ ] Code looks sound, good architectural decisions, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/)
- [ ] There is a Jira issue associated (note that "No-Issue" should be rarely used)
- [ ] Tests are included in `galaxy_ng/tests/integration` or `galaxy_ng/tests/functional`, and they fully cover necessary test scenarios… or tests not needed

### CODE REVIEW

**Review by himdel — APPROVED** [review]
LGTM :+1: 

(but the commit needs a `No-Issue` to pass checks)

File: galaxy_ng/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -138,20 +134,20 @@ msgstr "Récupérer l'importation de la collection"
 #: app/api/ui/viewsets/group.py:45
 #, python-format
 msgid "A group named %s already exists."
-msgstr "Un groupe nommé %s existe déjà."
+msgstr "Un groupe nommé  %s existe déjà."
```
Comment (himdel) [inline galaxy_ng/locale/fr/LC_MESSAGES/django.po:137]:
```suggestion
msgstr "Un groupe nommé %s existe déjà."
```

(the extra space is breaking tests)

**Comment by rooftopcellist** [comment]
Closing in favor of https://github.com/ansible/galaxy_ng/pull/1248

**Pull request closed without merging**